__tests__/Unit/Components/Tabs/Tab.test.tsx

```diff
@@ -0,0 +1,33 @@
+import Tabs from '@/components/Tabs';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TABS } from '@/components/tasks/constants';
+describe('Tabs Component', () => {
+  const onSelectMock = jest.fn();
+  it('should render all the buttons', () => {
+    render(<Tabs tabs={TABS} activeTab='ASSIGNED' onSelect={onSelectMock} />);
+    const presentTabs = screen.queryAllByRole('button');
+    expect(presentTabs.length).toBe(TABS.length);
+  });
+
+  it('check if selectTab() is called with right key', () => {
+    render(<Tabs tabs={TABS} activeTab='ASSIGNED' onSelect={onSelectMock} />);
+    const assignedBtn = screen.getByRole("button", { name: /ASSIGNED/i });
+    fireEvent.click(assignedBtn);
+    expect(onSelectMock).not.toHaveBeenCalledWith('COMPLETED');
```

Comment:
Line `16` assert doesn't make sense, you want to check the mock function called with which value?

Reply:
Like, if we click on Assigned button, it should have been called with 'assigned'

Reply:
Yes, `it should have been called with 'assigned'` so, the not-called-with assertion is not required.

Reply:
ok